packages/node_modules/@webex/plugin-meetings/src/reachability/request.js

```diff
@@ -0,0 +1,43 @@
+import {StatelessSparkPlugin} from '@ciscospark/spark-core';
+
+import {POST, GET, RESOURCE, API} from '../constants';
+
+/**
+ * @class RechabilityRequest
+ */
+class RechabilityRequest extends StatelessSparkPlugin {
+  /**
+   * gets the cluster information
+   * @returns {Promise}
+   */
+getClusters = () => this.request({
+  method: GET,
+  shouldRefreshAccessToken: false,
+  api: API.CALLIOPEDISCOVERY,
```

Comment:
these urls should probably be referenced in a config if they can be changed for integration/production etc. You'll want to give this a namespace to have access to the meetings plugin config

Reply:
The url is given by service catalog via the SDK so if we change the device url then the urls will change.